### ISSUE TITLE
Feature Request - Ability to type out entire strings

### ISSUE THREAD

**Issue by rbreaves** [mooz/xkeysnail#114]
There are some use cases where I think this would be helpful.

e.g. W("hello") vs K("h"),K("e"),K("l")... etc.

**Comment by joshgoebel** [comment]
Agree, wouldn't it be as simple as a small helper function `W` as you've named it?

**Comment by rbreaves** [comment]
I’d think so, either W or S for string.
